modules/ROOT/pages/building-https-proxy.adoc

```diff
@@ -3,62 +3,49 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-API Manager 2.x enables you to use the secrets you stored in your secret group to build an HTTPS-based API for your CloudHub or hybrid deployments. +
+API Manager 2.x enables you to use the secrets you stored in your secret group to build an HTTPS-based API for your CloudHub or hybrid deployments. 
 By configuring a TLS context for your inbound and outbound traffic, you can ensure your traffic's security.
 
-The secrets manager integration is only supported by API Proxies running Mule 4, and only for CloudHub, Hybrid, and standalone deployments. +
+The secrets manager integration is only supported by API Proxies running Mule 4, and only for CloudHub, Hybrid, and standalone deployments.
 Supported keystores for the TLS context are JCEKS, PKCS12, and JKS.
 
-If you want to build an HTTPS Proxy in Mule 3 or Runtime Fabric deployment target, follow the xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service] tutorial.
+To build an HTTPS Proxy in Mule 3 or Runtime Fabric deployment target, see xref:mule-runtime::build-an-https-service.adoc[Build an HTTPS Service].
```

Comment:
Add the: the Runtime Fabric deployment target?

Reply:
will fix